README.md

```diff
@@ -169,6 +169,7 @@ Perhaps with additional research, a workaround can be conceptualized that does n
 To run end-to-end tests, you must first have built the extension (`npm run build`).
 Then the Playwright tests can be run with `npx playwright test` for headless mode.
 If you want to run the tests interactively, you can use `npx playwright test --ui`.
+To mock the network requests from the service worker an environment variable must be set: PW_EXPERIMENTAL_SERVICE_WORKER_NETWORK_EVENTS=1.
```

Comment:
Can we put this all in a single script in `package.json`?

Reply:
yes, that is possible. will do

Reply:
Can you also update the README to explain to use `npm test`?

Reply:
sure. shall I also add a script to run the tests in UI mode / interactively?

Reply:
Sure.

Reply:
done